src/types.ts

```diff
@@ -13,4 +13,42 @@
 * under the License.
 */
 
-export type HttpMethod = "GET" | "POST";
\ No newline at end of file
+// HTTP requests & responses related 
+export type HttpMethod = "GET" | "POST";
+export interface ListCount {count: number}
```

Comment:
Rename types to be more specific, and move them to a types file in the page/component

Reply:
ok